docs/user_guide/bank/getting_started_partners.md

```diff
@@ -11,9 +11,9 @@ READY FOR REVIEW
 
 4/  The usual way to handle bringing on Partners to be able to make Requests is to invite them,  then have them fill in their profile before approving them, so that you get the information from them that your bank needs for grants, etc.   However, if that's not how you want to work,  it is also possible to invite and approve them in one step.
 
-5/  A lot of Banks set up a Partner as a proxy for their direct distribution to people in need.  This is allowed, and you can switch back and forth between being a Bank and a Partner with the same login
+5/  Many Banks set up a Partner as a proxy for their direct distribution to people in need.  This is allowed, and you can switch back and forth between being a Bank and a Partner with the same login.
```

Comment:
Note on Markdown formatting…I don't think it recognizes `5/` as anything semantic. If you use a `.` instead of a `/`, I believe it then generates html with `<li>` list items inside an overall `<ol>` ordered list.

It should also auto-fix any numbering typos, although that could sometimes be more confusing than helpful.

E.g., the following markdown
```markdown
1. First item
2. Second
2. Third
```

is formatted with correct numbering as

1. First item
2. Second
2. Third